pkg/kube/kubehttp.go

```diff
@@ -376,6 +390,91 @@ func (k *KubeHTTP) SetResource(ctx context.Context, resource Resource) error {
 	return nil
 }
 
+func (k *KubeHTTP) SetWegoConfig(ctx context.Context, config WegoConfig, namespace string) error {
+	configBytes, err := kyaml.Marshal(config)
+	if err != nil {
+		return fmt.Errorf("failed marshalling wego config: %w", err)
+	}
+
+	name := types.NamespacedName{Name: "weave-gitops-config", Namespace: namespace}
+
+	cm := &corev1.ConfigMap{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "ConfigMap",
+			APIVersion: corev1.SchemeGroupVersion.String(),
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name.Name,
+			Namespace: name.Namespace,
+		},
+	}
+
+	if err := k.Client.Get(ctx, name, cm); err != nil {
```

Comment:
Out of curiosity why not use the GetWegoConfig function here?

Reply:
I just wanna update the data field in the ConfigMap, the `GetWegoConfig` returns the parsed data, not the configmap itself